dev/com.ibm.ws.security.openidconnect.server/resources/WEB-CONTENT/clientAdmin/js/clientInputDialog.js

```diff
@@ -578,18 +592,16 @@ var clientInputDialog = (function() {
             "       <legend class='tool_modal_body_field_label'>" + fldName + "</legend>" +
             "       <div class='tool_modal_body_field_helper_text'>" + fldDescription + "</div>" +
             "       <div class='tool_modal_radio_button_horizontal_wrapper'>" +
-            "           <input id='" + fldId + "-true' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='true'";
-        field += fldDefault === 'true' ? " data-default='true'" : " data-default='false'";
-        field +=
-            "></input>" +
             "           <label for='" + fldId + "-true' class='tool_modal_radio_button_label'>" +
+            "               <input id='" + fldId + "-true' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='true'";
+        field += fldDefault === 'true' ? " data-default='true'" : " data-default='false'";
+        field += ">" +
             "               <span class='tool_modal_radio_button_appearance'></span>" + messages.TRUE + 
             "           </label>" +
-            "           <input id='" + fldId + "-false' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='false'";
-        field += fldDefault === 'false' ? " data-default='true'" : " data-default='false'";
-        field += 
-            "></input>" +
             "           <label for='" + fldId + "-false' class='tool_modal_radio_button_label'>" +
+            "               <input id='" + fldId + "-false' class='tool_modal_radio_button' type='radio' role='radio' aria-checked='false' name='" + fldId + "' value='false'";
+            field += fldDefault === 'false' ? " data-default='true'" : " data-default='false'";
+            field += ">" +
```

Comment:
\</input\>?

Reply:
No.....Not in HTML...from w3 schools....

<img width="648" alt="image" src="https://user-images.githubusercontent.com/29490139/70814791-d210c800-1d91-11ea-9035-764f81d04b17.png">